scripts/add-license.sh

```diff
@@ -16,32 +16,34 @@ LICENSE_TEMPLATE=$(cat <<EOF
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
 //
-//    http://www.apache.org/licenses/LICENSE-2.0
+//	http://www.apache.org/licenses/LICENSE-2.0
 EOF
 )
 
 # append all *.go in this repository with the LICENSE_TEMPLATE
 for file in $(find . -name "*.go" -type f); do
     	# skip files in the EXCLUDE_LIST
     	if [[ " ${EXCLUDE_LIST[@]} " =~ " ${file} " ]]; then
-		continue
+		    continue
     	fi
 
-    	# skip files that already have the LICENSE_TEMPLATE
-    	if grep -q "Copyright $YEAR The Gidari Authors." "${file}"; then
-		continue
-    	fi
+      # If the file starts with "// Copied from" then we don't want to prepend
+      # the license.
+      if grep -q "^// Copied from" "${file}"; then
+        continue
+      fi
 
-	# If the file starts with "// Copied from" then we don't want to prepend
-	# the license.
-	if grep -q "^// Copied from" "${file}"; then
-		continue
-	fi
+      sed -i '/^package/,$!d' "${file}"
```

Comment:
We don't need to do this. We should just update the script to add the extra line to avoid including copyrights in the package documentation.

Reply:
Hi @prestonvasquez, does this mean the other files in the root level would be fixed up manually?

Reply:
Added in https://github.com/alpstable/gidari/pull/384/commits/30e7ecba31c8e1486509efb1641edde9747a57bc

And also added newline to each go file, https://github.com/alpstable/gidari/pull/384/commits/22ea2373115bbc056e2bec13b9e620abcf0c2c87

Let me know if I shouldn't to the last commit